rvic/core/param_file.py

```diff
@@ -49,7 +49,7 @@ def finish_params(outlets, dom_data, config_dict, directories):
     # subset (shorten time base)
     if options['SUBSET_DAYS'] and \
             options['SUBSET_DAYS'] < routing['BASIN_FLOWDAYS']:
-        subset_length = (options['SUBSET_DAYS'] *
+        subset_length = int(options['SUBSET_DAYS'] *
                          SECSPERDAY / routing['OUTPUT_INTERVAL'])
```

Comment:
First of all, you should always use `isinstance` to [type check in python](https://stackoverflow.com/questions/152580/whats-the-canonical-way-to-check-for-type-in-python). 

Second, in this case, I think we should just wrap these in an `int` cast since it will just happen once in the init step (`gen_params_init()` I think). This will cover failure of the config reader of two cases ( `2 == int(2.0) == int('2')`). You'll want to raise an error if you get anything that can't be cast to an int (e.g. `int('foo')).